constants.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/lib/constants')
```

Comment:
@timneutkens 
We may need to add this file to `package.json` or may be inside our build file.
Could you have a look at that?

Reply:
Good point 👍